File: lab2/cpp/cpp/main.cpp
#include <iostream>
#include <fstream>
#include <string>
#include "Header.h"
#include <vector>
using namespace std;

int main() {
	vector<string> consoleText;
	vector<string> fileText;
	vector<string> changedFileText;
	vector<string> listOfTimes;

	// Ввод списка клиентов с клаватуры
	consoleText = readConsoleText(listOfTimes);

	// Запись текста в бинарный файл input.dot
	inputFileText(consoleText, "input.dot");

	// Чтение текста с бинарного файла
	fileText = readFileText();

	// Создание нового текста, который содержит нужные элементы
	changedFileText = changeText(fileText, listOfTimes);

	// Запись нового текста в бинарный файл output.dot
	inputFileText(changedFileText, "output.dot");

	// Вывод содержимого файла input.dot
	cout << "\nInput file text:\n";
	printFileText("input.dot");

	// Вывод содержимого файла output.dot
	cout << "\nOutput file text:\n";
	printFileText("output.dot");

	system("pause");
}

File: lab2/python/main.py
import func

# Ввод списка клиентов с клавиатуры
text, text_time = func.readConsoleText()

# Запись текста в бинарный файл input.dot
func.inputFileText(text, "input.dot")

# Чтение текста с бинарного файла
fileText = func.readFileText()

# Создание нового текста, который удовлетворяет условие
changedFileText = func.changeFileText(fileText, text_time)

# Запись нового текста в бинарный файл output.dot
func.inputFileText(changedFileText, "output.dot")

# Вывод содержимого бинарного файла input.dot
print("\nInput file text:")
func.printFileText("input.dot")

# Вывод содержимого бинарного файла output.dot
print("\nOutput file text:")
func.printFileText("output.dot")
